SQF/dayz_server/system/server_monitor.sqf

```diff
@@ -92,6 +92,7 @@ diag_log ("HIVE: Streamed " + str(_val) + " objects");
 	_storageMoney = _x select 9;
 
 	//set object to be in maintenance mode
+	_maintenanceMode = false;
```

Comment:
@icomrade @ebaydayz this fixes the undefined variable issue I was having, if there's a better fix feel free to alter this